app/views/shared/_request_status_information.html.erb

```diff
@@ -0,0 +1,37 @@
+<dl class='dl-horizontal dl-invert'>
+  <% if request.holdings.many? %>
+    <dt><%= t('forms.labels.holdings') %></dt>
+    <% request.holdings.each do |holding| %>
+      <dd><%= holding.callnumber %></dd>
+    <% end  %>
+  <% end %>
+
+  <% if request.destination.present? %>
+    <dt>
+      <%= label_for_pickup_libraries_dropdown(request.library_location.pickup_libraries) %>
+    </dt>
+    <dd>
+      <%= LibraryLocation.library_name_by_code(request.destination) %>
+    </dd>
+  <% end %>
+
+  <% if request.commentable? && request.data['comments'].present? %>
+    <dt><%= t("forms.labels.comments") %></dt>
+    <dd><%= request.data['comments'] %></dd>
+  <% end %>
+
+  <% if request.data.present? %>
```

Comment:
I honestly don't think they should be pulled into the request (but perhaps that's just my preference).  Different request objects may want to provide different pieces of data and this allows us to stay flexible in that respect without having to to lots of database and data migrations.